evosax/strategies/lm_ma_es.py

```diff
@@ -1,10 +1,10 @@
-from typing import Tuple, Optional, Union
+from typing import Optional, Tuple, Union
+
+import chex
 import jax
 import jax.numpy as jnp
-import chex
+from evosax.strategy import Strategy
```

Comment:
Can you change this back to the relative import? `from ..strategy import Strategy`

Reply:
thanks for your reviewing! I changed it in the latest commit.